pkg/policy/policy.go

```diff
@@ -1,4 +1,4 @@
-package config
+package policy
```

Comment:
can't comment on unchanged files but PolicyProvider -> Provider since the package name denotes the namespace i.e policy.Provider instead of policy.PolicyProvider